protos/sources/vega/events/v1/events.proto

```diff
@@ -317,23 +317,23 @@ message EpochEvent {
   uint64 seq = 1;
   // Action tells us what action is taking place
   vega.EpochAction action = 2;
-  // What time did this epoch start
+  // Time at which this epoch started
   int64  start_time = 3;
-  // What time should this epoch end
+  // Time at which this epoch should end
   int64  expire_time = 4;
-  // What time did it actually end
+  // Time at which this epoch did actually end
```

Comment:
```suggestion
  // Time at which this epoch actually ended
```

Reply:
I would have thought so. I've updated to reflect that.